pkgs/os-specific/linux/kloak/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "kloak";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "vmonaco";
+    repo = "kloak";
+    rev = "v${version}";
+    sha256 = "1jkds8i3x419fcrgdskzbwypgqbyqcypg6kris600l1y7z6hapj9";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./kloak ./eventcap $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Keystroke-level online anonymization kernel: obfuscates typing behavior at the device level";
+    longDescription = "Note that this requires loading the 'uinput' kernel module.";
```

Comment:
I'm not sure that is an appropriate use of the longDescription field.

Reply:
Perhaps not though I've seen basic usage notes in this field for other packages. Would a preferred alternative be a wrapper script to echo a warning if this kernel module is not loaded?

Reply:
I neither have a strong opinion nor much experience and would like to defer to someone else.